libs/@local/hash-isomorphic-utils/src/system-types/comment.ts

```diff
@@ -91,10 +91,10 @@ import {
   HasTextOutgoingLinkAndTarget,
   HasTextOutgoingLinksByLinkEntityTypeId,
   HasTextProperties,
-  Image,
-  ImageOutgoingLinkAndTarget,
-  ImageOutgoingLinksByLinkEntityTypeId,
-  ImageProperties,
+  ImageV1,
```

Comment:
`ImageV1` and `ImageV2` are both present because migration `009` updated `Image` to v2, but didn't update anything that referred to `Image`. So I think we need a clean-up migration that upgrades `Image`'s dependencies.

I think this is worth doing in this PR, because it is going to involve updating `User` (via has avatar / has cover image).

Reply:
I've addressed this in https://github.com/hashintel/hash/pull/4031/commits/e3a19b4ca2ecf5ec65f9463915f30cdd82f22976 as a drive by in the same migration file.

This was more work than I anticipated when I spoke to you earlier today, because it required updating the `Organization` entity type to `v2` which caused the `005` migration script to fail because it was trying to retrieve/create instances of the latest version of the `Organization` entity type, which doesn't exist at that point in the migrations. To resolve this I had to modify the `createOrg` method to allow for creating orgs of a previous version, and the `getOrgByShortname` method to get orgs via the base URL instead of the versioned URL.

In addition, because we were using the latest version of the organization type to determine the system account ID, I had to modify this in [/system-account.ts](https://github.com/hashintel/hash/pull/4031/commits/e3a19b4ca2ecf5ec65f9463915f30cdd82f22976#diff-5b0e2ab5f8b247e9bc0b2402dd7751c10f326b2514bc1ae3d0fe6e08ad391383) to explicitly look for the first version instead of the latest version of the organization, so that the system account ID can be determined before `v2` of the `Organization` type is created.